src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -214,6 +217,21 @@ public void teleopPeriodic() {
 
 		grabber.grab(driverstation.getGrabThrottle());
 
+		// Ramps state machines protects against conflicts
+		if (driverstation.getDeployButtonsDown()) {
+			ramps.deploy();
+		}
+
+		if (driverstation.getLeftRampButtonPressed()) {
+			ramps.leftLift();
+			ramps.leftDrop();
```

Comment:
Don't you want this to invert the state? Maybe make an a toggle() method, so Robot doesn't actually need to know if the ramps are up or down.